wpimath/generate_numbers.bzl

```diff
@@ -0,0 +1,171 @@
+__NUM_NUMS = 20
```

Comment:
The cmake python script could not be used here. Can probably be refactored to work with both